internal/librarian/generate.go

```diff
@@ -76,6 +76,10 @@ local working tree for inspection.`,
 	},
 }
 
+var libraryStateLoader = func(data []byte, libraryState *config.LibraryState) error {
```

Comment:
What is the value of having this higher order function? Why not just inline this code directly? I think that would make the code more readable.

Reply:
Done.